components/common.component.js

```diff
@@ -4,6 +4,7 @@ var Common = function () {
     this.feedbackClose = this.feedbackModal.element(by.css("button.btn.btn-white"));
 
     this.serverTime = element(by.binding("serverTime"));
+    this.stage = element.all(by.className("m-r-sm stage")).get(0);
```

Comment:
[Not according to styleguide](https://github.com/CarmenPopoviciu/protractor-styleguide#rule-07-prefer-protractor-locators-when-possible), but it will work.

Reply:
Ik heb naar een alternatief gezocht maar daarmee werkte het niet. Er is in het Frank!Framework een verschil tussen de "serverTime" en de "stage" text. In file topnavbar.html staat op regel 12:

<span class="m-r-sm text-muted serverTime">{{serverTime}}</span>

Op regel 15 staat:

<span class="m-r-sm stage">{{::otapStage}}</span>

Het verschil is de :: . Ik heb van Laurens begrepen dat je door de :: niet op binding otapStage kan zoeken.